test/over_react/util/class_names_test.dart

```diff
@@ -239,6 +239,65 @@ main() {
             }));
           });
         });
+
+        group('merge', () {
+          test('returns className merged from second builder instance', (){
```

Comment:
The "returns" part of this test description is misleading, since merge is a `void` and doesn't return anything.

Should be something like:
```dart
group('merge, () {
  test('merges the className from the provided builder instance', (){
```